samples/test/transfer-check.test.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const {assert} = require('chai');
+const {after, before, describe, it} = require('mocha');
+
+const {BucketManager, TransferJobManager, runSample} = require('./utils');
+
+describe('transfer-check', () => {
+  const testBucketManager = new BucketManager();
+  const testTransferJobManager = new TransferJobManager();
+
+  let projectId;
+  let transferJob;
+
+  before(async () => {
+    projectId = await testBucketManager.getProjectId();
```

Comment:
I noticed a lot of this setup/teardown is repeated for a lot of tests. Would it be possible to do as a root hook since you are using parallel mode?

Reply:
Good point - I had them separate to avoid race conditions (since shared buckets may make conflicting transfers) and to allow the tests to be ran individually (`npx mocha test/transfer-check.test.js`)